ashlar/filters.py

```diff
@@ -8,14 +8,24 @@
 from rest_framework.filters import BaseFilterBackend
 from rest_framework_gis.filterset import GeoFilterSet
 
-from ashlar.models import Boundary, Record, RecordType
+from ashlar.models import Boundary, Record, RecordSchema, RecordType
 
 
 class RecordFilter(GeoFilterSet):
 
+    record_type = django_filters.MethodFilter(name='record_type', action='filter_record_type')
+
+    def filter_record_type(self, queryset, value):
+        """ Method filter for records having a desired record type (uuid)
+
+        e.g. /api/records/?record_type=44a51b83-470f-4e3d-b71b-e3770ec79772
+
+        """
+        return queryset.filter(schema__in=RecordSchema.objects.filter(record_type=value))
```

Comment:
You should be able to chain double-underscores so that you can filter by a field on the related type: https://docs.djangoproject.com/en/1.8/topics/db/queries/#lookups-that-span-relationships